Assets/SEE/Utils/VersionControlSystems.cs

```diff
@@ -0,0 +1,144 @@
+using UnityEngine;
+using LibGit2Sharp;
+using System.Linq;
+using System.IO;
+
+/// <summary>
+/// Contains all version control systems as subclasses, which derives from IVersionControl,
+/// so that all have the same functionality.
+/// </summary>
+public class VersionControlSystems : MonoBehaviour
+{
+    /// <summary>
+    /// Implements the functionality of IVersionControl for Git.
+    /// </summary>
+    public class GitVersionControl : IVersionControl
+    {
+        /// <summary>
+        /// The name of the file, when it got deleted, or renamed.
+        /// </summary>
+        private string name = null;
+
+        /// <summary>
+        /// Returns the source code from the revision being compared against.
+        /// </summary>
+        /// <returns>the source code from the revision being compared against.</returns>
+        public string Show(string repositoryPath, string fileName, string commitID)
+        {
+            Repository repo = new(repositoryPath);
+            // Get file from commit
+            Blob blob = repo.Lookup<Blob>($"{commitID}:{Path.GetRelativePath(repositoryPath, fileName).Replace("\\", "/")}");
+
+            if (blob != null)
+            {
+                return blob.GetContentText();
+            }
+            else
+            {
+                // File got added and is only available in the other commit.
+                return "";
+            }
+        }
+
+        /// <summary>
+        /// Returns the source code from the revision, that gets compared.
```

Comment:
Comma is not needed. 

I don't understand the meaning of _that gets compared_. Who is compared against what? The documentation sound insufficient to me.